src/multi-selector.js

```diff
@@ -146,8 +146,7 @@ export default class MultiSelector extends React.Component {
     });
   };
 
-  removeItem = (item, e) => {
-    e.stopPropagation();
```

Comment:
hm, I may be wrong here, but wouldn't the better solution to be fixing it at line 76 (and keeping `stopPropagation`)?